cs251tk/toolkit/__main__.py

```diff
@@ -76,25 +95,37 @@ def main():
         logging.debug('Checking out {}'.format(date))
 
     if not os.path.exists("data"):
-        print('data directory not found', file=sys.stderr)
-        download = input("Download specs? (Y/N)")
-        if download and download.lower()[0] == "y":
-            repo = input("Which class? (SD/HD)")
-            if repo and repo.lower()[0] == 's':
-                with chdir(basedir):
-                    run(['git', 'clone', 'https://github.com/StoDevX/cs251-specs.git', 'data'])
-                    if not args['stogit']:
-                        stogit_url = compute_stogit_url(course="sd", stogit=None, _now=datetime.date.today())
-            elif repo and repo.lower()[0] == "h":
-                with chdir(basedir):
-                    run(['git', 'clone', 'https://github.com/StoDevX/cs241-specs.git', 'data'])
-                    if not args['stogit']:
-                        stogit_url = compute_stogit_url(course="hd", stogit=None, _now=datetime.date.today())
+        if args['ci']:
+            if args['course']:
+                url = download_specs(args['course'], basedir, args['stogit'])
+                if not args['stogit']:
+                    stogit_url = url
             else:
-                print("Class not recognized", file=sys.stderr)
+                print("data directory not found and no course specified")
                 sys.exit(1)
+
         else:
-            sys.exit(1)
+            print('data directory not found', file=sys.stderr)
+            if args['course']:
+                download = input("Download specs for {}? (Y/N)".format(args['course'].upper()))
+                if download and download.lower()[0] == "y":
```

Comment:
I think you can move this  line and the input line out of the conditional

Reply:
The difference is that it has a slightly different prompt if `course` is not defined. (R118)

Reply:
Ah. Ok.